src/test/java/org/jenkinsci/plugins/credentialsbinding/impl/UsernamePasswordBindingTest.java

```diff
@@ -52,7 +58,7 @@
         FreeStyleProject p = r.createFreeStyleProject();
         p.getBuildWrappersList().add(new SecretBuildWrapper(Collections.<Binding<?>>singletonList(new UsernamePasswordBinding("AUTH", c.getId()))));
         p.getBuildersList().add(new Shell("set +x\necho $AUTH > auth.txt"));
-        r.configRoundtrip(p);
+        r.configRoundtrip((Item)p);
```

Comment:
ambiguous method call.

`FreeStyleProject` is an `Item` as well as a `Project` so the call is [ambiguous](https://github.com/jenkinsci/jenkins/blob/7adbeef5a70a37768f6d2f8caf3f9e29c8e57436/test/src/main/java/org/jvnet/hudson/test/JenkinsRule.java#L1041-L1049)

the code will not compile in eclipse without this - I thought I removed this call from Jenkins Rule a while ago and was just using an older version - but perhaps not.